.circleci/config.yml

```diff
@@ -128,6 +128,11 @@ jobs:
       - run:
           name: Install publish dependencies
           command: sudo pip install -U awscli
+      - run:
+          name: Override AWS Credentials
+          command: |
+            echo 'export AWS_ACCESS_KEY_ID="$LAYER_AWS_ACCESS_KEY_ID"' >> $BASH_ENV
```

Comment:
Checking -- I don't think so, but making sure this won't accidentally echo the creds somehow in CircleCI? (which would be bad because the project is public)

Reply:
The `>> $BASH_ENV` redirects to bash, so no it won't echo. Circle CI docs here: https://circleci.com/docs/2.0/env-vars/

Reply:
Helpful, thank you!